gitea/docker-compose.yml

```diff
@@ -31,7 +31,7 @@ services:
       GITEA__database__PASSWD: "moneyprintergobrrr"
 
   db:
-    image: mariadb:10.5.12@sha256:dfcba5641bdbfd7cbf5b07eeed707e6a3672f46823695a0d3aba2e49bbd9b1dd
+    image: mariadb:10.11.2@sha256:dd0f492b6b6e7bb4aa707181b799d4efe42cb3a9f6012ec3dbaf326d402151e8
```

Comment:
Using a newer DB version will require careful testing to ensure this version of MariaDB works with existing installations that would have been using version `10.5.12`.

Reply:
@JimSB and @nevets963 I have tested this on existing gitea installations (amd64 and arm64) and had no issues related to the mariadb update. All my repos were maintained, and I was able to push new commits.